Makefile

```diff
@@ -658,6 +660,14 @@ test_versionable: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --profile $(CARGO_PROFILE) \
 		-p tfhe-versionable
 
+.PHONY: ci_test_backward_compatibility
+ci_test_backward_compatibility: install_rs_build_toolchain
```

Comment:
ci is generally a suffix as far as I can tell in the Makefile 😅

Reply:
looks like we have both style 🙄

Reply:
I put it at the end and uniformized the other ones in another comit

Reply:
thanks